RockLib.Diagnostics/DefaultTraceListener.cs

```
@@ -38,21 +38,24 @@ private void Write(string message, bool useWriteLine)
         {
             if (NeedIndent)
                 WriteIndent();
-            try
+
+            if (!string.IsNullOrWhiteSpace(LogFileName))
             {
-                using (var stream = new FileInfo(LogFileName).Open(FileMode.OpenOrCreate))
-                using (var streamWriter = new StreamWriter(stream))
+                try
+                {
+                    using (var stream = new FileInfo(LogFileName).Open(FileMode.OpenOrCreate))
+                    using (var streamWriter = new StreamWriter(stream))
+                    {
+                        stream.Position = stream.Length;
+                        if (useWriteLine)
+                            streamWriter.WriteLine(message);
+                        else
+                            streamWriter.Write(message);
+                    }
+                }
+                catch
```

Intentional empty catch?

---

Yeah. `Debug.WriteLine` wasn't doing anything anyway when compiled as release.